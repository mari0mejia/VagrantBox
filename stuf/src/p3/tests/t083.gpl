// use std::numbers::pi from #include<numbers> (as of C++20)
// DO NOT CREATE YOUR OWN VALUE FOR PI
double d = atan( 7 );
double e = atan( 12.5 );
string g = atan( 7 );
string h = atan( 12.5 );
