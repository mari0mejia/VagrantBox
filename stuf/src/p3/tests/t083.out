gpl.cpp::main()
  input file(tests/t083.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double d = 81.8699
double e = 85.4261
string g = "81.869898"
string h = "85.426079"
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
